presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -83,6 +85,9 @@
 {
     private static final Logger log = Logger.get("TestQueries");
 
+    private static final DateTimeFormatter timeWithUtcZoneFormat = DateTimeFormatter.ofPattern("HH:mm:ss.SSS 'UTC'"); // UTC zone would be printed as "Z" in "XXX" format
+    private static final DateTimeFormatter timeWithZoneOffsetFormat = DateTimeFormatter.ofPattern("HH:mm:ss.SSS XXX");
```

Comment:
I guess there is no DataTimeFormatter which takes multiple patterns? I did not find anything...

Reply:
There is, `java.time.format.DateTimeFormatterBuilder#appendPattern`. I tried but couldn't use it, as "utc zone format" is used to parse `LocalTime` and the other is used to parse `OffsetTime`.